subsys/net/lib/nrf_cloud/src/nrf_cloud_rest.c

```diff
@@ -73,8 +73,10 @@ LOG_MODULE_REGISTER(nrf_cloud_rest, CONFIG_NRF_CLOUD_REST_LOG_LEVEL);
 #define API_GET_PGPS_BASE		API_VER API_LOCATION "/pgps"
 
 #define API_DEVICES_BASE		"/devices"
-#define API_DEVICES_STATE_TEMPLATE	API_VER API_DEVICES_BASE "/%s/state"
-#define API_DEVICES_MSGS_TEMPLATE	API_VER API_DEVICES_BASE "/%s/messages"
+#define API_DEVICES_TEMPLATE		API_VER API_DEVICES_BASE "/%s"
+#define API_DEVICES_TRANSFORM_TEMPLATE	API_DEVICES_TEMPLATE "?transform=%s"
+#define API_DEVICES_STATE_TEMPLATE	API_DEVICES_TEMPLATE "/state"
```

Comment:
What about other places in this file that use API_DEVICES_STATE_TEMPLATE and API_DEVICES_MSGS_TEMPLATE that expect there to be a `%s`?

Reply:
it is included in `API_DEVICES_TEMPLATE` which is: `"/devices/%s"`

Reply:
right, missed that thanks